pallets/runtime/src/bridge.rs

```diff
@@ -157,6 +163,15 @@ decl_storage! {
         /// become unlocked. Pending transactions are also included here if they will be tried automatically.
         TimelockedTxs get(fn timelocked_txs):
             map hasher(twox_64_concat) T::BlockNumber => Vec<BridgeTx<T::AccountId, T::Balance>>;
+
+        /// limit on bridged POLYX per identity for the testnet. (POLYX, LIMIT_REST_BLOCK)
```

Comment:
Maybe instead of the X_Y format you could describe the parameters in an itemized list? E.g.,
```
# Parameters
* POLYX amount,
* ...
```
That would also improve formatting in the resulting HTML.
 
Just looking at the current comment it's not clear what those made up names mean.